src/components/Map/components/FileDragNDrop.tsx

```diff
@@ -66,3 +74,30 @@ const FileDragNDrop = () => {
 };
 
 export default FileDragNDrop;
+
+const isLambert93Projection = (geojson: any) => geojson.crs?.properties?.name && geojson.crs.properties.name.includes('2154');
+
+const convertCoordinates = (coords: any, from: string, to: string): any => {
+  if (Array.isArray(coords[0])) {
+    return coords.map((coord: any) => convertCoordinates(coord, from, to));
+  }
+  return proj4(from, to, coords);
+};
+
+const convertGeoJSON = (geojson: any): any => {
```

Comment:
convert to what?

Reply:
convertLambert93GeoJSONToWGS84 :joy:  J'ai ajouté des commentaires aussi. L'idée est d'obtenir un geojson en WGS 84. J'ai refactoré aussi un peu cf autre commentaire.